main.c

```diff
@@ -332,6 +332,12 @@ static void usage(const char *argv0)
         "caps except CAP_NET_BIND_SERVICE if running as the root)\n");
     printf("--enable-seccomp         enable seccomp to limit syscalls "
            "(experimental)\n");
+    printf("--outbound-addr=IPv4     sets outbound ipv4 address to bound to "
+           "(experimental)\n");
+    printf("--outbound-addr6=IPv6    sets outbound ipv6 address to bound to "
+           "(experimental)\n");
+    printf("--disable-dns            disables 10.0.2.3 to host dns redirect "
```

Comment:
Not always 10.0.2.3

Reply:
yes, well I am not sure  what comment should be there. Is it better in latest or any suggestions ?